hrui/js/src/civihr-popup/civihr-popup.js

```diff
@@ -4,14 +4,61 @@ Prevent Popups to overflow tables.
 (function ($) {
   'use strict';
 
-  // .btn-slide is a continer of popup and a button triggering it
-  $('.btn-slide').attrchange(function (e) {
-    var $body = $('body');
+  $('body').on('click', '.btn-slide', function (e) {
```

Comment:
Can we use `event` or `ev` instead of `e` since `e` can also mean "error"?

Reply:
yes, that's a good observation, though this variable is not being used, so I'll remove it.